language/opt/run_clm.py

```diff
@@ -321,24 +322,32 @@ def main():
         logger.warning("You are instantiating a new config instance from scratch.")
     logger.info("Model config is created")
 
-    if args.model_name_or_path == 'facebook/13b':
+    if args.model_name_or_path == 'facebook/opt-13b':
         tokenizer = GPT2Tokenizer.from_pretrained(args.model_name_or_path)
```

Comment:
I think models larger than 13B will require GPT2 Tokenizer as well.

Reply:
Acctually, it is not a problem with GPT2Tokenizer, since all models use GPT2Tokenizer.
This is a problem only for opt-13b, opt-13b can't use fast tokenizer.
It is fine for 30b.